mds/migrations/0044_compliance.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.1.11 on 2019-09-09 14:34
```

Comment:
You should give a even clearer name : 
0044_add_compliance_model.py so that next time someone edits compliance model, he would be able to call the migration `xxxx_add_compliance_<field_name>.py` or so